### ISSUE TITLE
Finetuning of dnb using gaussian-rectified rbm

### ISSUE THREAD

**Issue by snurkabill** [mihaelacr/pydeeplearn#9]
Hello, 

I would like to ask if your library supports chaining gaussian-rectified rbms into dbn and then finetuning whole think with stochastic gradient descent? I am new at python so I don't understand well your code.

**Comment by mihaelacr** [comment]
Hi. I will close this issue as I replied to you via email.

**Comment by snurkabill** [comment]
yes, thanks. 

btw, I've started actively work with your library :)
